carRental/src/components/CheckBox/CheckBox.jsx

```diff
@@ -1,17 +1,15 @@
 import React from "react";
-
+import "./CheckBox.css";
 const Checkbox = ({ label, isSelected, onCheckboxChange }) => (
   <div className="form-check">
-    <label>
-      <input
-        type="checkbox"
-        name={label}
-        checked={isSelected}
-        onChange={onCheckboxChange}
-        className="form-check-input"
-      />
-      {label}
-    </label>
+    <label className="check-label">{label}</label>
+    <input
+      type="checkbox"
+      name={label}
+      checked={isSelected}
+      onChange={onCheckboxChange}
+      className="form-check-input"
```

Comment:
Can we change this thingy to checkbox-input instead?

Reply:
done